plugins/gatsby-plugin-openapi/gatsby-node.js

```diff
@@ -0,0 +1,114 @@
+  }
+
+  if (content === null) {
+    return;
+  }
+  try {
+    const parseContent = () => {
+      try {
+        return JSON.parse(content);
+      } catch (error) {
+        return content;
+      }
+    };
+
+    const parsedContent = parseContent();
+
+    var data =
+      parsedContent.paths &&
+      Object.keys(parsedContent.paths).reduce((acc, apiPath) => {
+        let result = Object.entries(parsedContent.paths[apiPath]).map(
+          ([method, rest]) => {
+            const methodPath = parsedContent.paths[apiPath][method];
+
+            let readableUrl = methodPath["operationId"]
+              .replace(/(?:(the|a|an) +)/g, "")
+              .trim()
+              .replace(/\s/g, "-")
+              .toLowerCase();
+
+            let responses =
+              (methodPath["responses"] &&
+                Object.entries(methodPath["responses"]).map(
+                  ([status_code, responses_rest]) => ({
+                    ...responses_rest,
+                    content:
+                      (responses_rest.content &&
+                        Object.entries(responses_rest.content).reduce(
+                          (acc, [content_type, content_values]) => {
+                            Object.entries(content_values).map(
+                              ([k, v]) => (acc[k] = JSON.stringify(v, null, 2))
+                            );
+                            acc["content-type"] = content_type;
+                            return acc;
+                          },
+                          {}
+                        )) ||
+                      null,
+                    status_code,
+                  })
+                )) ||
+              null;
+
+            let requestBody =
+              (methodPath["requestBody"] && {
+                ...methodPath["requestBody"],
+                content:
+                  (methodPath["requestBody"]["content"] &&
+                    Object.entries(methodPath["requestBody"]["content"]).reduce(
+                      (acc, [content_type, content_values]) => {
+                        Object.entries(content_values).map(
+                          ([k, v]) => (acc[k] = JSON.stringify(v, null, 2))
+                        );
+                        acc["content-type"] = content_type;
+                        return acc;
+                      },
+                      {}
+                    )) ||
+                  null,
+              }) ||
+              null;
+
+            return {
+              ...rest,
+              method,
+              apiPath,
+              responses,
+              requestBody,
+              readableUrl,
+            };
+          }
+        );
+
+        acc.push(...result);
+        return acc;
+      }, []);
+
+    const rootNode = {
+      ...parsedContent,
+      paths: data,
+      id: createNodeId(`openAPI-root`),
+      children: [],
+      parent: null,
+      internal: {
+        contentDigest: createContentDigest("openAPI"),
```

Comment:
should the digest be hashing the data? 

here's how ive done it:

https://github.com/getsentry/sentry-docs/pull/2119/files#diff-c03e26a205ea6e8099f39ca510c6f61dR220-R236